api/server.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const server = express();
+const db = require("./db.js");
+
+server.use(express.json());
+
+let checkIFExists = element => {
+  return db.filter(obj => obj.title === element).length > 0;
+};
+
+// endpoints
+server.get("/games", (req, res) => {
+  res.status(200).json(db);
+});
+
+server.post("/games", (req, res) => {
+  let { title, genre, releaseYear } = req.body;
```

Comment:
Why do you destructure `releaseYear` here when you don't use it?

Reply:
honestly I thought I was going to use it, but then never did, lol.